inspectkv/inspectkv.go

```diff
@@ -64,6 +64,43 @@ func GetDDLInfo(txn kv.Transaction) (*DDLInfo, error) {
 	return info, nil
 }
 
+// GetDDLJobs returns the DDL jobs and an error.
+func GetDDLJobs(txn kv.Transaction) ([]*model.Job, error) {
+	t := meta.NewMeta(txn)
+	cnt, err := t.DDLJobQueueLen()
+	if err != nil {
+		return nil, errors.Trace(err)
+	}
+
+	jobs := make([]*model.Job, cnt)
+	for i := range jobs {
+		jobs[i], err = t.GetDDLJob(int64(i))
```

Comment:
Is DDLJobQueueLen and GetDDLJob in an atomic operation or snapshot?
Will DDLJobQueueLen change during this line?